google/cloud/apigee/registry/applications/v1alpha1/registry_styleguide.proto

```diff
@@ -98,16 +104,19 @@ message Rule {
         (google.api.field_behavior) = REQUIRED
     ];
 
-    // Severity of violating this rule.
     enum Severity {
-        ERROR = 0;
-        WARNING = 1;
-        INFO = 2;
-        HINT = 3;
+        UNSPECIFIED = 0;
+        ERROR = 1;
+        WARNING = 2;
+        INFO = 3;
+        HINT = 4;
```

Comment:
As above, the unspecified field should be `SEVERITY_UNSPECIFIED` and it would be nice to have comments on the type and these values explaining the different severities.